plugins/org.eclipse.elk.core/src/org/eclipse/elk/core/util/LoggedGraph.java

```diff
@@ -108,10 +112,14 @@ public String getTag() {
      * Serializes the graph to a string for it to be saved to a file.
      */
     public String serialize() {
+	// TODO what is the effect?
+	// elkjs-exclude-start
         graphType.isTypeCompatible(graph);
```

Comment:
There's no effect here. I think I expected the method to throw an exception. I changed the implementation to throw an exception when constructing a new LoggedGraph instance.